src/chimera/instruments/astelco.py

```diff
@@ -94,8 +103,8 @@ def __init__(self):
                 self._calibration[rate][direction] = 1
 
         self._user = "admin"
-        self._password = "admin"
-        self._aahost = "localhost"
+        self._password = "admin" 
+        self._aahost = "192.168.10.3"
```

Comment:
yes, it should indeed.

On Wed, Jan 28, 2015 at 3:55 PM, William Schoenell <notifications@github.com
> wrote:

> In src/chimera/instruments/astelco.py
> <https://github.com/astroufsc/chimera/pull/31#discussion_r23706826>:
>
> > @@ -94,8 +103,8 @@ def __init__(self):
> >                  self._calibration[rate][direction] = 1
> >
> >          self._user = "admin"
> > -        self._password = "admin"
> > -        self._aahost = "localhost"
> > +        self._password = "admin"
> > +        self._aahost = "192.168.10.3"
>
> This part should be on the config file, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astroufsc/chimera/pull/31/files#r23706826>.
>